doc/portfile.7

```diff
@@ -590,59 +638,67 @@ Location to store downloaded distfiles.
 .Sy Default:
 .Em ${portdbpath}/distfiles/${dist_subdir}/
 .It Ic os.arch
-Identifies hardware type (e.g. "powerpc").
+Identifies hardware type (e.g.,
+.Dq powerpc ) .
```

Comment:
This has a different quoting from some instances of `powerpc` mentioned above (where it was underlined) and different from some other quoted variables (with a single quote) written above. Is there a special reason for that?

Reply:
I don't see another quoted instance of `powerpc`.  Which line do you mean?  And where do you see single quotes?